release.json

```diff
@@ -168,4 +175,4 @@
         "JMXFETCH_VERSION": "0.19.0", 
         "JMXFETCH_HASH": "28f345c2d407e85802c43201e1e1ea5b8664f39ac705ac8e6b44295b641c5d70"
     }
-}
\ No newline at end of file
+}
```

Comment:
I will remove this before merge.

Reply:
You can remove it from this PR and create a separate branch from this commit, where you would then add this. Then that other branch can be used for deploying to our internal infrastructure, leaving this PR clean and ready for review.